test/Microsoft.AspNetCore.SignalR.Tests/TestClient.cs

```diff
@@ -52,7 +60,7 @@ public TestClient(bool synchronousCallbacks = false)
                 Application.Out.TryWrite(memoryStream.ToArray());
             }
         }
-
+        
```

Comment:
In my editor there is single line freespace as in other places across the file. Not sure why it shows line added there.